Assets/HoloToolkit/Utilities/Scripts/SphereBasedTagalong.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// A Tagalong that stays at a fixed distance from the camera and always
+    /// seeks to stay on the edge or inside a sphere that is straight in front of the camera.
+    /// </summary>
+    public class SphereBasedTagalong : MonoBehaviour
+    {
+        /// <summary>
+        /// Sphere radius
```

Comment:
NIT: Please convert these to ToolTip.
I have heard feedback from folks that tool tips work better than summary comments in editor.